lib/DynamicEntryPlugin.js

```diff
@@ -36,6 +49,11 @@ class DynamicEntryPlugin {
 		compiler.hooks.make.tapAsync(
 			"DynamicEntryPlugin",
 			(compilation, callback) => {
+				/**
```

Comment:
@sandersn @weswigham expand to lines 38-44 where the `typeof` type error occurs

Reply:
You're passing in the constructor where an instance is expected.

Reply:
How would I type the original reference if we do want to expect the constructor? I think maybe this is where I misunderstood.